plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/cache/TestCachingHiveMetastore.java

```diff
@@ -660,14 +712,77 @@ private static void await(CountDownLatch latch, long timeout, TimeUnit unit)
         }
     }
 
+    static class PartitionCachingAssertions
+    {
+        private final CachingHiveMetastore cachingHiveMetastore;
+        private final MockThriftMetastoreClient thriftClient;
+        private Consumer<CachingHiveMetastore> sutAction = hiveMetastore -> {};
```

Comment:
Is there is any expansion for `sutAction` ?

Reply:
sut is a shortcut for `system under test`. It's a way to mark this as a holder for all actions that are actually being tested. I could rename it to `cachingHiveMetastoreInteractions` or `metastoreInteractions/Actions`. For me sut is ok, but maybe the fact that you need to know what it means makes it worst than one of the above.

Reply:
> sut is a shortcut for `system under test`.

would never guess